### ISSUE TITLE
Feedback if generated file returns "empty"

### ISSUE THREAD

**Issue by falfren** [echa-ecm/report_generator_scripts#2]
Hello everyone,

I was wondering if there was an option to get some feedback via console (or else) if the uploaded `*.ftl` template file for report generation was faulty and no proper report could be generated.

Currently, it looks like the script will upload a given `*.ftl` and initiate the report generation procedure without asking.
In my case, the expected csv file will be empty and I do not have clue for what reason.
The only way to find out is to get into IUCLID and start the report generation manually. If the `*.ftl` the report should be generated from is faulty, the file will not be generated and a huge red exclamation mark will indicate the error. 
After that I will be able to download a `*.txt` which will provide further information on the error.  
Those information - in my opinion - could also be provided via an REST API call if the report generation fails instead of just an empty report.

Thanks in advance for helping me out on this ...

**Comment by echa-ecm** [comment]
Hi @falfren  - we indeed rely on the server/error logs to find issues with the FTLs. From v6.6.19.0 of IUCLID, there is an error log available via the User Interface to get error information. 

Are you requesting that this error log be served via the REST API? Or something else, sorry I don't quite understand the question.

**Comment by falfren** [comment]
Good morning @echa-ecm 

you rephrased my request for the better. This is exactly what I am wishing for. Serving the error log via the REST API request.
Is there a way to do that in an easy way? Maybe an option that has to be checked?

My feeling is, that the result of an unsuccessful report generation process shouldn't be an empty file but the error log itself.

**Comment by echa-ecm** [comment]
Morning @falfren - as this is a useful discussion for the community at large, I am transferring the question to the Customisation forum, in which I have broken down your question into two parts: https://community.oecd.org/message/58421#58421

**Comment by falfren** [comment]
Very nice. Thank you for doing so.
Just an addition:
I finished the docker container swarm for the report generator development environment and was testing the setup.
Just now I found out, that there is an error returned if one wants to generate a report from a template that has not been created in the IUCLID instance. Maybe there is a way to further elaborate on this initial step?

But this finding can be moved to the customisation forum as well.
